transport/http2_server.go

```diff
@@ -400,6 +409,17 @@ func (t *http2Server) handleWindowUpdate(f *http2.WindowUpdateFrame) {
 	}
 }
 
+func (t *http2Server) handleGoAway(f *http2.GoAwayFrame) {
+	t.goaway = true
+	for id, _ := range t.activeStreams {
```

Comment:
I do not think we need to iterate activeStreams because server does not initiate any streams in grpc protocol.